src/dependencyTreeProvider.ts

```
@@ -0,0 +1,133 @@
+
+            this.command = {
+                command: 'layered-gen.openFile',
+                title: 'ファイルを開く',
+                arguments: [filePath]
+            };
+        }
+    }
+}
+
+export class DependencyTreeProvider implements vscode.TreeDataProvider<DependencyItem> {
+    private _onDidChangeTreeData: vscode.EventEmitter<DependencyItem | undefined | null | void> = new vscode.EventEmitter<DependencyItem | undefined | null | void>();
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)